api/app/models/settings.rb

```diff
@@ -17,6 +17,8 @@ class Settings < ApplicationRecord
       social_share_message: "Shared from Manifold Scholarship",
       library_disabled: false,
       restricted_access: false,
+      restricted_access_heading: "Access to this project is restricted.",
+      restricted_access_body: "Only users granted permission may view this project's texts, resources, and other content.",
```

Comment:
I would enhance this model in a couple small ways. We should define these strings as constants, and add accessors for them for use in the project.

```
class Settings < ApplicationRecord
  DEFAULT_RESTRICTED_ACCESS_HEADING = "Access to this project is restricted.".freeze
  DEFAULT_RESTRICTED_ACCESS_BODY = ""Only users granted permission may view this project's texts, resources, and other content.".freeze

  # Add these constants in the defaults hash where you have the strings, then, later in the class:

  # @return [String]
  def default_restricted_access_heading
    general.fetch(:restricted_access_heading, DEFAULT_RESTRICTED_ACCESS_HEADING)
  end

  # @return [String]
  def default_restricted_access_body
    general.fetch(:restricted_access_body, DEFAULT_RESTRICTED_ACCESS_BODY)
  end
end